src/cpu/aarch64/jit_generator.hpp

```diff
@@ -360,7 +360,9 @@ class jit_generator : public Xbyak_aarch64::CodeGenerator, public c_compatible {
 public:
     jit_generator(void *code_ptr = nullptr, size_t code_size = MAX_CODE_SIZE,
             bool use_autogrow = true)
-        : Xbyak_aarch64::CodeGenerator(code_size, code_ptr) {}
+        : Xbyak_aarch64::CodeGenerator(code_size,
+                (code_ptr == nullptr && use_autogrow) ? Xbyak_aarch64::AutoGrow
+                                                      : code_ptr) {}
```

Comment:
One thing to keep in mind with enabling autogrow feature is that it impacts how you can define multiple entry-points in the same binary. Basically, you need to make sure that a. you store each entry-point offset (and not their address) and b. you should make sure that autogrow does not impact the binary alignment (should not be the case by default).

Reply:
@mgouicem 
Thank you for the advice. I'll keep it in my mind.